### ISSUE TITLE
delegate from vesting account

### ISSUE THREAD

**Issue by okwme** [chainapsis/keplr-extension#63]
i'm getting this error when trying to delegate from a new vesting account:
```
Account's address is unknown: {"type":"cosmos-sdk/ContinuousVestingAccount","value":{"base_vesting_account":{"base_account":{"address":"xxxxxx","account_number":"193682"},"original_vesting":[{"denom":"uatom","amount":"xxxxxxx"}],"delegated_free":[],"delegated_vesting":[],"end_time":"1646089200"},"start_time":"1614365510"}}
```

**Comment by litvintech** [comment]
@okwme Maybe it's related to cosmos-sdk's [#8601](https://github.com/cosmos/cosmos-sdk/issues/8601)

**Comment by dogemos** [comment]
It's probably related to this issue(https://github.com/chainapsis/keplr-extension/issues/60). It's getting fixed (https://github.com/chainapsis/keplr-extension/commit/10aa47fc7cae0470fb50549209db616263763534), and we're aiming for a release tomorrow.

**Comment by dogemos** [comment]
We've just released v0.8.0 which should have resolved this issue. Can you check and see if everything works?
